src/components/Navbar/Drawer/Drawer.module.scss

```diff
@@ -13,17 +12,19 @@
   overflow-y: auto;
   overflow-x: hidden;
   overscroll-behavior-y: contain;
+  top: 0;
+  bottom: 0;
   @include breakpoints.tablet {
     width: constants.$side-menu-desktop-width;
   }
   &.left {
-    inset-inline-start: -100%;
+    left: -100%;
```

Comment:
We're using css logical properties btw, so `left` will automatically becoems `right` on RTL https://css-tricks.com/css-logical-properties/

Reply:
I got it but those props does not supported by most iOS devices. Which makes those props useless unless it would be supported